packages/mobile/android/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I know this file looks weird, it's because of `SafetyNet`. It depends on `react-native-securerandom` which doesn't support automatic linking, and when doing it manually it threw an error because this file was missing. 

This manual linking is the reason for the changes to the `settings.gradle` and the Podfile as well.

Reply:
Would love to fork/fix/upstream after wednesday as good citizens of the open source world

Reply:
Yeah, I like doing that sort of stuff too, but the maintainers don't seem very active. The first reason I had to fork was to add support for AndroidX since it was failing for that reason. There's a PR that adds that from almost a year ago: https://github.com/rajivshah3/react-native-google-safetynet/pull/283 which has this message:

> Thanks for the PR! I can review and investigate the build failure in a few days when I have some time

(December 2019)